enterprise/models.py

```diff
@@ -2,42 +2,108 @@
 """
 Database models for enterprise.
 """
-
 from __future__ import absolute_import, unicode_literals
 
-# from django.db import models
-from django.utils.encoding import python_2_unicode_compatible
+import uuid
 
+from simple_history.models import HistoricalRecords  # likely a bug in import order checker
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
 from model_utils.models import TimeStampedModel
 
 
+class EnterpriseCustomerManager(models.Manager):
+    """
+    Model manager for Enterprise Customer model.
+
+    Filters out inactive Enterprise Customers, otherwise works the same as default model manager.
+    """
+
+    # This manager filters out some records, hence according to the Django docs it must not be used
+    # for related field access. Although False is default value, it still makes sense to set it explicitly
+    # https://docs.djangoproject.com/en/1.10/topics/db/managers/#base-managers
+    use_for_related_fields = False
+
+    def get_queryset(self):
+        """
+        Return a new QuerySet object. Filters out inactive Enterprise Customers.
+        """
+        return super(EnterpriseCustomerManager, self).get_queryset().filter(active=True)
+
+
 @python_2_unicode_compatible
 class EnterpriseCustomer(TimeStampedModel):
     """
-    TODO: replace with a brief description of the model.
+    Enterprise Customer is an organization or a group of people that "consumes" courses.
+
+    Users associated with an Enterprise Customer take courses on the edX platform.
+
+    Enterprise Customer might be providing certain benefits to their members, like discounts to paid course
+    enrollments, and also might request (or require) sharing learner results with them.
+
+    Fields:
+        short_code (UUIDField, PRIMARY KEY): Enterprise Customer code - used to reference this Enterprise Customer in
+            `                                other parts of the system (SSO, ecommerce, analytics etc.)
+        name (CharField): Enterprise Customer name
+        active (BooleanField): used to mark inactive Enterprise Customers - implements "soft delete" pattern
     """
 
-    # TODO: add field definitions
+    class Meta(object):
+        verbose_name = _("Enterprise Customer")
+        verbose_name_plural = _("Enterprise Customers")
+
+    objects = models.Manager()
+    active_customers = EnterpriseCustomerManager()
+
+    short_code = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
Field should be named uuid. Using it as a primary key may have a slight performance degradation; but, depending on your scale it may not be enough to matter.